about.html

```diff
@@ -10,56 +10,77 @@
   <link rel="stylesheet" href="css/index.css">
 
 </head>
-
+<section>
 <body>
     <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+      <header>
+        <div class="headLinks">
+        
+        <img src="https://github.com/SierraFenstemaker/web-Sprint-Challenge-User-Interface-and-Git/blob/master/img/lambda-black.png?raw=true" alt="lambda name"/>
+    
+        
+        
+            <a href="index.html">Home</a>
+                <a href="about.html">About</a>
+                <a href="#">Products</a>
+                <a href="#">Blog</a>
+                <a href="#">Contact</a>
+        
+    </div>
+    </header>
+    <img src="https://github.com/SierraFenstemaker/web-Sprint-Challenge-User-Interface-and-Git/blob/master/img/jumbo-about.png?raw=true" alt="city"/>
+    <div class="about">
```

Comment:
The content area of the page is a great place to use semantic `<section> and/or <article>` elements.